app/models/mixins/supports_feature_mixin.rb

```diff
@@ -73,15 +73,13 @@ module SupportsFeatureMixin
     :cinder_service             => 'Cinder storage service',
     :create_floating_ip         => 'Floating IP Creation',
     :create_host_aggregate      => 'Host Aggregate Creation',
-    :create_network_router      => 'Network Router Creation',
```

Comment:
@durandom, thanks for pointing this. Some more clean-up to do, I'll remove the [WIP] once the dependent PR is merged.